cmd/ebrelayer/relayer/ethereum.go

```diff
@@ -323,6 +324,9 @@ func (sub EthereumSub) CheckNonceAndProcess(txFactory tx.Factory,
 			}
 			events = append(events, event)
 		}
```

Comment:
We should add a comment here explaining why we are sorting the events

Reply:
Done